docs/using-radicle/creating-patches.md

```diff
@@ -0,0 +1,97 @@
+---
+id: creating-patches
+title: Creating patches
+---
+
+To make a contribution to another user's project, you must first find it on the
+network. To find a project, you need its [**Radicle ID**][ri]. This can be found
+on the project page under the project name, or in a seed node dashboard such as
+[sprout.radicle.xyz](http://sprout.radicle.xyz).
+
+![Identity][id]
+
+Once you have the Radicle ID, copy it into the Upstream search bar. From there,
+you can follow the project to track it. Once it is found in your network of
+connected peers, it will replicate the project to your machine. You won't see
+any information on the project until it is verified by the protocol.
+
+![Search bar][sb]
+
+![Found Project][fp]
+
+### Forking projects
+
+To contribute to another user's project you have to fork it forst.Using the
+**Fork** option in the Upstream client will clone the project to a specified
+location on your local machine and publish your version to the Radicle network
+as a new project.
+
+![Fork Project][fo]
+
+ Forking a project will move it from your **Following** tab to your **Projects**
+tab. This is because once you fork a project, you create your own view of it
```

Comment:
```suggestion
tab. This is because once you fork a project, you create your own copy of it
```

“View” sounds more like something that I cannot update and that nobody else sees.